### PR TITLE
Make it able to build both 1.4 and 1.5 e2e images

### PR DESCRIPTION
**Pull request by krzyzacy**
Pretty much the same code change as test-images, and I'd like to start things with this change and figure out how to properly use different images.

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/test-infra/972)

### CODE REVIEW

**Comment by krzyzacy** [comment]
I'll make a non-dirty image and change e2e-runner after this.

File: jenkins/e2e-image/Dockerfile-1.4 [hunk 1]
```diff
@@ -15,7 +15,7 @@
 # This file creates a build environment for building and running kubernetes
 # unit and integration tests
 
-FROM gcr.io/google-containers/kubekins-test:v20160822
+FROM gcr.io/k8s-testimages/kubekins-test:1.4-v20161007-49d47fe
```
Comment (ixdy) [inline jenkins/e2e-image/Dockerfile-1.4]: do we want to have two separate `Dockerfile`s?

or do we just want to parameterize the `FROM` line to source from the correct `kubekins-test` image, and then keep everything else the same?

anyone on @kubernetes/test-infra-admins have thoughts?

Comment (krzyzacy) [inline jenkins/e2e-image/Dockerfile-1.4]: I am actually more towards combine e2e-images and test-images dir to avoid those duplications.

**Comment by krzyzacy** [comment]
/close

I guess I'll just make a different one

**Pull request closed without merging**